Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1184,19 +1160,53 @@ def get_test_list_and_content_packs_to_install(files_string, branch_name, two_be
     return tests, packs_to_install
 
 
-def create_filter_envs_file(tests, two_before_ga, one_before_ga, ga, conf, id_set):
+def get_from_version_and_to_version_from_modified_files(all_modified_files_paths: set, id_set: dict) -> Tuple[str, str]:
+    """
+    computes the lowest from version of the modified files and the highest to version of the modified files
+    If no limitation is found- default will be 0.0.0 - 99.99.99
+    Args:
+        all_modified_files_paths: All modified files
+        id_set: the content of the id.set_json
+
+    Returns:
+        The string representation of the lowest from version and highest to version
+    """
+    max_to_version = LooseVersion('0.0.0')
+    min_from_version = LooseVersion('99.99.99')
+    for _, artifacts in id_set.items():
+        for artifact_dict in artifacts:
+            for _, artifact_details in artifact_dict.items():
+                if artifact_details.get('file_path') in all_modified_files_paths:
+                    from_version = artifact_details.get('fromversion')
+                    to_version = artifact_details.get('toversion')
+                    if from_version:
+                        min_from_version = min(min_from_version, LooseVersion(from_version))
+                    if to_version:
+                        max_to_version = max(max_to_version, LooseVersion(to_version))
+    if max_to_version.vstring == '0.0.0':
+        max_to_version = LooseVersion('99.99.99')
+    if min_from_version.vstring == '99.99.99':
+        min_from_version = LooseVersion('0.0.0')
+    logging.debug(f'modified files are {all_modified_files_paths}')
```

Comment:
I think it's important to show what are the changed files it takes into consideration, and what are the resulting selected tests and packs.

I think it could help reveal issues with this process if there are some, and that's why it's under debug mode.